addshare/addshare.c

```diff
@@ -37,15 +37,15 @@ static void usage(int status)
 {
 	int i;
 
-	fprintf(stderr,
+	g_printerr(
```

Comment:
Could you elaborate why we have to change these with g_print or g_printerr?

Reply:
It's one less parameter.